guides/k8-akka-cluster/src/main/paradox/index.md

```diff
@@ -0,0 +1,465 @@
+
+Next we will transform the generated Docker `ENTRYPOINT` instruction:
+
+```
+dockerCommands :=
+  dockerCommands.value.flatMap {
+    case ExecCmd("ENTRYPOINT", args @ _*) => Seq(Cmd("ENTRYPOINT", args.mkString(" ")))
+    case v => Seq(v)
+  }
+```
+
+The script above transforms `ENTRYPOINT` from `exec` form to `shell` form. The `shell` form is required to ensure the environment variables declared in the `dockerEntrypoint` argument is sourced from the shell within the Docker container. Refer to Docker [ENTRYPOINT](https://docs.docker.com/engine/reference/builder/#entrypoint) documentation for the difference between `exec` and `shell` form.
+
+The container image will be published to the the default Docker repository specified by SBT Native Packager. To publish to a different repository, set the `dockerRepository` setting to the repository you wish to publish. Please note this is an optional step. The Kubernetes deployment in this guide expects the image to be located at `mygroup/myapp`. To match the repository we need to set the `dockerRepository` to `mygroup`:
+
+```
+dockerRepository := Some("mygroup")
+```
+
+When the image is published, the version tag will be derived from the SBT project version. However it's possible to also publish both the `latest` tag and the version tag by enabling the `dockerUpdateLatest` setting. Please note this is an optional step. The Kubernetes deployment in this guide expects the `latest` image tag to be present. To match this expectation we need to enable `dockerUpdateLatest` setting:
+
+```
+dockerUpdateLatest := true
+```
+
+Additional SBT setting documentation to control the Docker image build process is available at the [Docker Plugin](http://www.scala-sbt.org/sbt-native-packager/formats/docker.html?highlight=dockercommand) documentation page from SBT Native Packager.
+
+#### Using a smaller Docker base image (optional)
+
+_This is an optional step._
+
+SBT Native Packager by default uses [openjdk:latest](https://hub.docker.com/_/openjdk/) image from Docker Hub.
+
+To reduce the image size and the container startup time in Kubernetes, it's possible to use a base image with smaller footprint.
+
+This is accomplished by deriving an image from [openjdk:8-jre-alpine](https://hub.docker.com/_/openjdk/) with `bash` installed as the Docker entrypoint script generated by SBT Native Packager requires `bash` to be present.
+
+Create the image using the following `Dockerfile`:
+
+```
+FROM openjdk:8-jre-alpine
+RUN apk --no-cache add --update bash coreutils
+```
+
+Once created, this image can be used as a base image by specifying the `dockerBaseImage` setting in the `build.sbt`:
+
+```
+dockerBaseImage := "<my image name>"
+```
+
+### Publishing to Docker registry - Maven
+
+We will be using the [fabric8](https://dmp.fabric8.io/) Maven plugin to containerize the application.
```

Comment:
In new line; We explain the setup depending on if you use a multi-module or single module project, refer to the respective section below

?